### ISSUE TITLE
Product single: Inconsistent behavior upon clicking the social sharing buttons.  Twitter opens in a new window while the rest opens in a new tab.

### ISSUE THREAD

**Issue by roseliesromeo** [titan-cebu/DVA-Fabrications#29]
I believe social sharing buttons should open in a new window just like below
![image](https://cloud.githubusercontent.com/assets/11186313/14058723/efad45f6-f364-11e5-8673-943869f3209a.png)

**Comment by roseliesromeo** [comment]
2 of the social media opens in new tab while the rest in another window

**Comment by jsemblantetitanweb** [comment]
I've tried going to the support of the addthis plugin and there was a thread on it that says "this is the behavior of there plugin and they are still developing it into a more consistent behavior" which the ticket says. I can't fix this one right now since it is the plugin issue and Also I've tried the other plugin which is the sharethis and it also behaves the same way as the addthis plugin. Currently I am using the addthis plugin since it is use by most of the developers.
